Cargo.toml

```diff
@@ -22,13 +22,12 @@ members = [
     "maybe-rayon",
     "mersenne-31",
     "monolith",
-    "multi-stark",
     "poseidon",
     "poseidon2",
     "reed-solomon",
     "rescue",
     "symmetric",
-    "tensor-pcs",
```

Comment:
should I try and fix tensor-pcs or delete it for now?

Reply:
Yeah I'm good with deleting, can always restore it from history if it's useful later (though little progress was made anyway)